### PR TITLE
[Doc] Add sphinx basis 

### PR DESCRIPTION
**Pull request by Herklos**
https://octobot-channels.readthedocs.io/

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# Optionally build your docs in additional formats such as PDF and ePub
+formats: all
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+  version: 3.8
+  install:
+    - requirements: requirements.txt
+    - requirements: dev_requirements.txt
```
Comment (GuillaumeDSM) [inline .readthedocs.yml]: missing newline :)

File: dev_requirements.txt [hunk 2]
```diff
@@ -15,3 +15,6 @@ setuptools
 wheel
 
 pur
+
+sphinx==2.4.4
+sphinx_rtd_theme
```
Comment (GuillaumeDSM) [inline dev_requirements.txt:20]: 👍

**Comment by GuillaumeDSM** [comment]
> https://octobot-channels.readthedocs.io/

is it up to date ? it looks empty

<blockquote><div><strong><a href="https://octobot-channels.readthedocs.io/">Welcome to Read the Docs — OctoBot-Channels latest documentation</a></strong></div></blockquote>

**Comment by Herklos** [comment]
> > https://octobot-channels.readthedocs.io/
> 
> is it up to date ? it looks empty
> 
> > **[Welcome to Read the Docs — OctoBot-Channels latest documentation](https://octobot-channels.readthedocs.io/)**

Yes it's up to date, it's only sphinx basis. You can use the search bar to search for Class and method names or code directly. There is no manual documentation for now.

<blockquote><div><strong><a href="https://octobot-channels.readthedocs.io/">Welcome to Read the Docs — OctoBot-Channels latest documentation</a></strong></div></blockquote>

**Comment by Herklos** [comment]
> > > https://octobot-channels.readthedocs.io/
> > 
> > 
> > is it up to date ? it looks empty
> > > **[Welcome to Read the Docs — OctoBot-Channels latest documentation](https://octobot-channels.readthedocs.io/)**
> 
> Yes it's up to date, it's only sphinx basis. You can use the search bar to search for Class and method names or code directly. There is no manual documentation for now.
> 
> > **[Welcome to Read the Docs — OctoBot-Channels latest documentation](https://octobot-channels.readthedocs.io/)**

Sorry, you were able to do that. It's seems that it doesn't work anymore.

<blockquote><div><strong><a href="https://octobot-channels.readthedocs.io/">Welcome to Read the Docs — OctoBot-Channels latest documentation</a></strong></div></blockquote>

**Comment by Herklos** [comment]
> > > > https://octobot-channels.readthedocs.io/
> > > 
> > > 
> > > is it up to date ? it looks empty
> > > > **[Welcome to Read the Docs — OctoBot-Channels latest documentation](https://octobot-channels.readthedocs.io/)**
> > 
> > 
> > Yes it's up to date, it's only sphinx basis. You can use the search bar to search for Class and method names or code directly. There is no manual documentation for now.
> > > **[Welcome to Read the Docs — OctoBot-Channels latest documentation](https://octobot-channels.readthedocs.io/)**
> 
> Sorry, you were able to do that. It's seems that it doesn't work anymore.

It works on the ['docs-add' version](https://octobot-channels.readthedocs.io/en/docs-add/octobot_channels.channels.html).
You also have the module list [here](https://octobot-channels.readthedocs.io/en/docs-add/py-modindex.html)

**Review by GuillaumeDSM — APPROVED** [review]
💯 10

**Pull request merged**